stock_warehouse_flow/views/stock_warehouse_flow.xml

```diff
@@ -0,0 +1,154 @@
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+          <div class="oe_title">
+              <label for="name" class="oe_edit_only" />
+              <h1><field name="name" placeholder="Flow Name" /></h1>
+          </div>
+          <group>
+            <group name="info" colspan="4" col="4">
+              <field
+                                name="warehouse_id"
+                                attrs="{'readonly': [('delivery_route_id', '!=', False)]}"
+                            />
+              <field name="active" invisible="1" />
+            </group>
+            <group name="from" string="From">
+              <field name="from_picking_type_id" />
+              <div
+                                colspan="2"
+                                attrs="{'invisible': [('from_picking_type_id', '=', False)]}"
+                            >
+                <span style="width: 4em;" />
+                =>
+                <field name="from_location_src_id" nolabel="1" />
+                to
+                <field name="from_location_dest_id" nolabel="1" />
+              </div>
+              <field name="carrier_ids" widget="many2many_tags" />
+              <field
+                                name="move_domain"
+                                widget="domain"
+                                options="{'model': 'stock.move', 'in_dialog': true}"
+                            />
+            </group>
+            <group name="to" string="To">
+              <field
+                                name="delivery_steps"
+                                attrs="{'invisible': [('delivery_route_id', '!=', False)]}"
+                            />
+              <field
+                                name="sequence_prefix"
+                                attrs="{'invisible': [('delivery_route_id', '!=', False)], 'required': [('delivery_route_id', '=', False)]}"
+                            />
+              <field
+                                name="delivery_route_id"
+                                attrs="{'invisible': [('delivery_route_id', '=', False)]}"
+                            />
+              <field
+                                name="to_picking_type_id"
+                                attrs="{'invisible': [('delivery_route_id', '=', False)]}"
+                            />
+              <button
+                                name="action_generate_route"
+                                type="object"
+                                string="Generate route"
+                                class="oe_highlight"
+                                attrs="{'invisible': [('delivery_route_id', '!=', False)]}"
+                            />
+            </group>
+          </group>
+          <notebook>
+            <page string="Rules" name="rules">
+              <field name="rule_ids" nolabel="1" />
```

Comment:
Display from the rules:
* action
* operation type
* location from
* location to
* fixed procurement group (group_id)